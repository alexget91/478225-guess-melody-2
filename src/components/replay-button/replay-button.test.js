import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";
import ReplayButton from "./replay-button";

it(`Replay button correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<BrowserRouter>
      <ReplayButton text={`1`} onClick={jest.fn()}/>
    </BrowserRouter>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
