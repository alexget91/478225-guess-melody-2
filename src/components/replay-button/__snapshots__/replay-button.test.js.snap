// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Replay button correctly renders after relaunch 1`] = `
<a
  className="replay"
  href="/"
  onClick={[Function]}
>
  1
</a>
`;
